src/humancellatlas/data/metadata/api.py

```diff
@@ -582,8 +583,9 @@ class ImagingPreparationProtocol(Protocol):
     pass
 
 
-@dataclass
+@dataclass(init=False)
 class ManifestEntry:
+    json: MutableMapping = field(repr=False)  # would be MutableJSON if azul types were available
```

Comment:
```suggestion
    # FIXME: would be MutableJSON if Azul types were available
    #        <ticket url>
    json: MutableMapping = field(repr=False)
```

Add ticket reference. Azul is proper noun.